modules/core/include/opencv2/core/types.hpp

```diff
@@ -545,7 +545,8 @@ class CV_EXPORTS RotatedRect
      */
     RotatedRect(const Point2f& point1, const Point2f& point2, const Point2f& point3);
 
-    /** returns 4 vertices of the rectangle
+    /** returns 4 vertices of the original up-right(angle=0) rectangle with the same center and size.
```

Comment:
This is not correct, it returns vertices of rotated rectangle, not an original. It might be better to note somehow that the "top", "bottom", "left", "right" specifiers take (or don't take???) rotation into account or give an example where the first vertex in the returned array will be located at the bottom left corner of 0-degree rectangle and in the top right corner of 180-degree rectangle.

BTW, for non-upright rectangles terms bottomLeft, etc. do not have meaning.

Reply:
> BTW, for non-upright rectangles terms bottomLeft, etc. do not have meaning.

That's why I refer to the original up-right rectangle since these names are misleading when paired with a rotated one.

```suggestion
    /** returns 4 vertices of the rotated rectangle
```